server/lib/date-adjust.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const fs = require("fs");
+
+// Set a variable for the current epoch date
+// Initialize it to 3 days before now
+// 86400000ms === 1 Day, 1 Day * 3 === 259200000ms
+let adjustedDate = Date.now() - 259200000;
+
+// Increment by one day (86400000 milliseconds), per tweet
+// to ensure the tweets remain in order and staggered by a day
+function dayDecrement() {
+  adjustedDate += 86400000;
```

Comment:
Can we refactor out these "magic" numbers to variables?

```javascript
const SECONDS_IN_DAY = 86400000;
```

Reply:
Done, refactored out a lot of useless code, but I'm an a cross:
I've converted the messy 86.4 million ms to a cleaner `1000 * 60 * 60 * 24 * days`, and I've completely removed the requirement for the `adjustedDate` and `decrementDay` functions.

Should I declare `1000 * 60 * 60 * 24` as a variable called `day` or because it's one line is it better to leave it as is?

I'm now instead of creating an `adjustedDay` variable, just removing days from Date.now() via the `map()` index.  Is that kosher or considered hacky in nature?